projects/extension/tests/vectorizer/test_vectorizer.py

```diff
@@ -193,6 +203,7 @@ def test_vectorizer_timescaledb():
         db_url("test"), autocommit=True, row_factory=namedtuple_row
     ) as con:
         with con.cursor() as cur:
+            cur.execute("set statement_timeout = '5s'")
```

Comment:
why is this added?

Reply:
It's related to a non resolved issue with that test locally,
for some reason the SQL call to the run_job fails, 
and at least this way it fails fast, not after a few minutes.

Reply:
I thought it used to fail with the server running too, but that's correct. 
removing this statement now.
thanks!